html-forms/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Forms</title>
+  </head>
+
+  <body>
+    <form>
+      <p> Email
```

Comment:
This should be a `label` element

Reply:
Also, please remove the `p` element holding all the form content, it is unnecessary 👍

Reply:
![image](https://user-images.githubusercontent.com/69921839/105093706-05735b00-5a58-11eb-9bba-0398062dfb67.png)